### PR TITLE
Implement new pipeline interface

### LINKED ISSUE #5: Update CC example with new dataset and pipeline interface

### PR DESCRIPTION
**Pull request by GeorgesLorre**
Resolves: https://github.com/ml6team/fondant-usecase-filter-creative-commons/issues/5

### CODE REVIEW

**Comment by RobbeSneyders** [comment]
Thanks @GeorgesLorre!

Can you run the following command and push the changes? It excludes the output from your notebook.
```
git config filter.strip-notebook-output.clean 'jupyter nbconvert --ClearOutputPreprocessor.enabled=True --to=notebook --stdin --stdout --log-level=ERROR' 
```

You should only have to run it once to configure it for the repo.

**Comment by RobbeSneyders** [comment]
LGTM, apart from one missing change.

Can you change:
```python
run_explorer_app(
    base_path=BASE_PATH,
    container="fndnt/data_explorer",
    tag="latest",
    port=8501
)
```
to
```python
run_explorer_app(base_path=BASE_PATH)
```
We made a change in this release so automatically the matching explorer version is used.

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
-fondant==0.7.0
+fondant==0.8.dev4
```
Comment (PhilippeMoussalli) [inline requirements.txt]: Shouldn't we base ourselves on the latest official non-dev release?

File: README.md [hunk 2]
```diff
@@ -40,15 +42,10 @@ fondant run local pipeline.py
...
-
-```bash
-fondant run local filter_pipeline
-```
 
 After the pipeline is succeeded you can explore the data by using the fondant data explorer:
 
 ```bash
-fondant explore --base_path ./data
+fondant explore --base_path ./fondant-artifacts
```
Comment (PhilippeMoussalli) [inline README.md]: Can we called it `data-dir` to stay consistent with the other notebooks?

File: src/pipeline.py [hunk 3]
```diff
@@ -1,46 +1,53 @@
...
-
-logger = logging.getLogger(__name__)
-sys.path.append("../")
+import pyarrow as pa
+from fondant.pipeline import Pipeline
 
 PIPELINE_NAME = "cc-image-filter-pipeline"
 PIPELINE_DESCRIPTION = "Load cc image dataset"
-BASE_PATH = "./data-dir"
+BASE_PATH = "./fondant-artifacts"
```
Comment (PhilippeMoussalli) [inline src/pipeline.py]: Same comment here as above

**Review by PhilippeMoussalli — APPROVED** [review]
Thanks Georges! Left a few small comments

File: requirements.txt (hunk 1 above)
Comment (GeorgesLorre) [reply requirements.txt]: Yes, this was created before the release. Will update.

**Pull request merged**